### PR TITLE
Teams app | Resource hub

### PR DESCRIPTION
**Pull request by v-haalam**
All the help you need to use Teams, all in one place. Get up and running with just the basics, dive into popular topics, master features designed for your work style or industry, get answers to frequently asked questions, and see what’s new.

Client app pages designed in react js.
Backend code integration developed in c# and .net core 6 framework.

### CODE REVIEW

**Review by Wajeed-msft — CHANGES_REQUESTED** [review]
1. Rename the folder as  Microsoft-Teams-Samples/samples/**msteams-application-resourcehub**
2. Update the [ReadMe file](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/v-haalam/resourcehub/samples/app-resourcehub/csharp/README.md) with documentation, deployment guide, solution overview. Ex: [Microsoft-Teams-Samples/samples/msteams-application-qbot](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/msteams-application-qbot#qbot). Also include screenshot which will showcase your solution. 
3. Add metadata for sample code discovery:
	a. Add page type YML metadata
Ex: [Microsoft-Teams-Samples/samples/msteams-application-qbot at main · OfficeDev/Microsoft-Teams-Samples (github.com)](https://github.com/OfficeDev/Microsoft-Teams-Samples/tree/main/samples/msteams-application-qbot)

image
b. Include sample.json under assets folder.
Ex: [Microsoft-Teams-Samples/sample.json at main · OfficeDev/Microsoft-Teams-Samples (github.com)](https://github.com/OfficeDev/Microsoft-Teams-Samples/blob/main/samples/msteams-application-qbot/assets/sample.json)

**Comment by v-haalam** [comment]
> sample

Resolved all comments

**Review by Wajeed-msft — APPROVED** [review]

**Review by vin-vija — APPROVED** [review]

File: samples/msteams-application-resourcehub/README.md [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+products:
+- office-teams
+- office
+- office-365
+languages:
+- csharp
+- typescript
+extensions:
+ contentType: samples
+ createdDate: "19-09-2022 12:00 PM"
```
Comment (Wajeed-msft) [inline samples/msteams-application-resourcehub/README.md:13]:
```suggestion
 createdDate: "10/10/2022 02:56:00 PM"
```

**Review by Wajeed-msft — APPROVED** [review]

**Review by Prithvi-MSFT — APPROVED** [review]
Approved

**Pull request merged**